packages/cozy-client/src/CozyClient.js

```diff
@@ -1592,6 +1600,15 @@ instantiation of the client.`
     })
   }
 
+  /**
+   * At any time put an error function
+   *
+   * @param {Function} [onError] - Set a callback for queries which are errored
+   */
+  setOnError(onError) {
+    this.options.onError = onError
```

Comment:
I'm wondering if you should not test if onError is already defined and thrown an error or log.warn / error? In order to avoid to set several times?

Reply:
Yes it would make sense to throw an error, we don't really need to want to set it several times